omegaconf/basecontainer.py

```diff
@@ -412,6 +412,7 @@ def _set_item_impl(self, key: Any, value: Any) -> None:
         )
 
         def wrap(key: Any, val: Any) -> Node:
+            print(key, val)
```

Comment:
Do not commit this.
I also recommend that you use a debugging instead of printing to debug things.

Reply:
oops